### PR TITLE
License upgrade doc

### PR DESCRIPTION
**Pull request by benradstone**

### CODE REVIEW

File: docs/quickstarts/operation/license_upgrade.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+id: license_upgrade
+title: Upgrade your License
+sidebar_label: License Upgrade
+---
+
+You can upgrade your Fusion license through the UI by following these steps.
```
Comment (catrionabarker) [inline docs/quickstarts/operation/license_upgrade.md:7]: I think we need to very briefly include where you might get a license from.

Comment (catrionabarker) [inline docs/quickstarts/operation/license_upgrade.md:7]: Maybe a slightly more detailed intro? HIghlighting that you can have a new license already or you can request/buy a new one via the UI.

File: docs/quickstarts/operation/license_upgrade.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+sidebar_label: License Upgrade
+---
+
+You can upgrade your Fusion license through the UI by following these steps.
+
+1. Click the **Upgrade License** option on the dashboard.
+1. If you do not have a new license, click on the option to purchase a license now. Follow the provided instructions to obtain your license.
+
+   Once your license has been sent, ensure it is available on your local machine.
+1. Upload your new license via the Drag 'n' drop or clicking to select files.
```
Comment (catrionabarker) [inline docs/quickstarts/operation/license_upgrade.md:13]: I think it should be click not clicking

File: docs/quickstarts/operation/license_upgrade.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+1. Review the license preview and ensure it is correct before selecting to **Upload**.
+
+   The License information will then update to your new limits and expiry date.
```
Comment (catrionabarker) [inline docs/quickstarts/operation/license_upgrade.md:16]: Update with rather than to?

File: docs/quickstarts/operation/license_upgrade.md (hunk 1 above)
Comment (benradstone) [reply docs/quickstarts/operation/license_upgrade.md]: Reworded this so that it puts this point across.

**Pull request merged**